packages/styles/scss/theme/_foundation.scss

```diff
@@ -61,6 +61,12 @@
 
   --ilo-color-purple: rgba(150, 10, 85, 1);
 
+  --ilo-color-ux-notification-icon: var(--ilo-color-white);
+  --ilo-color-ux-notification-type-error: var(--ilo-color-red);
+  --ilo-color-ux-notification-type-info: var(--ilo-color-blue);
+  --ilo-color-ux-notification-type-success: var(--ilo-color-green);
+  --ilo-color-ux-notification-type-warning: var(--ilo-color-yellow);
```

Comment:
I think we can get rid of ux prefix

Reply:
and also, `icon` doesn't need to have color definition, just reference the color directly for that one

Reply:
Sure @GGKapanadze